executor/aggregate.go

```diff
@@ -1031,18 +1058,18 @@ func (e *HashAggExec) execute(ctx context.Context) (err error) {
 	}
 }
 
-func (e *HashAggExec) spillUnprocessedData(isFullChk bool) (err error) {
-	if isFullChk {
-		return e.listInDisk.Add(e.childResult)
-	}
+func (e *HashAggExec) spillUnprocessedData() (err error) {
 	for i := 0; i < e.childResult.NumRows(); i++ {
-		e.tmpChkForSpill.AppendRow(e.childResult.GetRow(i))
-		if e.tmpChkForSpill.IsFull() {
-			err = e.listInDisk.Add(e.tmpChkForSpill)
+		groupKey := e.groupKeyBuffer[e.childResult.GetRow(i).Idx()]
+		partitionIdx := farm.Hash32(groupKey) % defPartitionsToBeSpilled
```

Comment:
Hash32 can only calcuated is the first round. 
Other round will process the data in the same partition, so they hash32 values is same, and will re-spilled to the same partition.  So we don't calc it again.